domain/model/state/types.go

```diff
@@ -300,16 +300,18 @@ func (info *dbModelUserInfo) toModelUserInfo() (coremodel.ModelUserInfo, error)
 }
 
 type dbReadOnlyModel struct {
-	UUID              string `db:"uuid"`
-	ControllerUUID    string `db:"controller_uuid"`
-	Name              string `db:"name"`
-	Type              string `db:"type"`
-	Cloud             string `db:"cloud"`
-	CloudType         string `db:"cloud_type"`
-	CloudRegion       string `db:"cloud_region"`
-	CredentialOwner   string `db:"credential_owner"`
-	CredentialName    string `db:"credential_name"`
-	IsControllerModel bool   `db:"is_controller_model"`
+	UUID              string         `db:"uuid"`
+	ControllerUUID    string         `db:"controller_uuid"`
+	Name              string         `db:"name"`
+	OwnerName         string         `db:"owner_name"`
+	OwnerUUID         string         `db:"owner_uuid"`
+	Type              string         `db:"type"`
+	Cloud             string         `db:"cloud"`
+	CloudType         string         `db:"cloud_type"`
+	CloudRegion       sql.NullString `db:"cloud_region"`
+	CredentialOwner   sql.NullString `db:"credential_owner"`
+	CredentialName    string         `db:"credential_name"`
```

Comment:
This should be a nullable string as well. Credential owner and name are either both set or both not set